server/images.go

```diff
@@ -702,18 +702,33 @@ func convertModel(name, path string, fn func(resp api.ProgressResponse)) (string
 	return path, nil
 }
 
+func modelFilepath(n model.Name) (string, error) {
+	if !n.IsFullyQualified() {
+		return "", fmt.Errorf("unqualified model name")
+	}
+	return filepath.Join(n.Host, n.Namespace, n.Model, n.Tag), nil
+}
+
 func CopyModel(src, dst model.Name) error {
 	manifests, err := GetManifestPath()
 	if err != nil {
 		return err
 	}
 
-	dstpath := filepath.Join(manifests, dst.FilepathNoBuild())
```

Comment:
`Filepath` is a useful method since name.Model is used to reference files on disk very often. See https://github.com/ollama/ollama/pull/3715 https://github.com/ollama/ollama/pull/3718 https://github.com/ollama/ollama/pull/3737

Reply:
Added back.